### ISSUE TITLE
Cannot add item

### ISSUE THREAD

**Issue by ColinWeiss** [Tea-Mod-Loader/Tea#2]
Always show get texture error

**Comment by steviegt6** [comment]
Could you please elaborate further?

**Comment by steviegt6** [comment]
[Also consider joining the Discord server so it's easier to communicate.](https://discord.com/invite/qrZ4Bpz)

**Comment by ColinWeiss** [comment]
It say :"The given keyword is not in the dictionary"...
![Error](https://user-images.githubusercontent.com/69356100/95869410-e81a5b00-0d9d-11eb-9bd4-dda86020c5ca.png)

**Comment by steviegt6** [comment]
Do you have a .png image with the exact same name in the exact same directory? Capitalization matters, by the way.

**Comment by ColinWeiss** [comment]
![Error2](https://user-images.githubusercontent.com/69356100/95930083-31979400-0df8-11eb-9cff-69dcfe348f29.png)
![My VS](https://user-images.githubusercontent.com/69356100/95930088-33f9ee00-0df8-11eb-9863-9bc6d4d1e377.png)
![My VS2](https://user-images.githubusercontent.com/69356100/95930089-352b1b00-0df8-11eb-822a-f290ee3ab013.png)

Look...It puzzles me...I am a Modder——2 Years.I won't make such a small mistake...

**Comment by ColinWeiss** [comment]
When I say "look", I mean to show you the pictures.I'm not good at English.Sorry.

**Comment by ColinWeiss** [comment]
Is too slow to visit foreign websites......I don't know what to say for my Wifi

**Comment by steviegt6** [comment]
Really weird, I'll look into it later today.

**Comment by ColinWeiss** [comment]
Don't be tired! But I can load examplemod normally,It's strange.

**Comment by ColinWeiss** [comment]
Don't be tired!Pay attention to your body.

**Comment by steviegt6** [comment]
Haha, I said "weird", not "tired". Don't worry about me. :^)

**Comment by steviegt6** [comment]
Oh, it might be an issue with the path, try moving your item and sprite to the root folder of your mod (and change the namespace as well, obviously). If that's the case I'll have to do some debugging with that.

**Comment by ColinWeiss** [comment]
I tried.I moved the texture to the root directory, but it still doesn't work. I found that it seems that the top-level namespace and mod class have the same name.

**Comment by ColinWeiss** [comment]
Is the cause of the problem?

**Comment by steviegt6** [comment]
Most likely.

**Comment by ColinWeiss** [comment]
![！！！](https://user-images.githubusercontent.com/69356100/95932815-6824dd00-0dff-11eb-8ad9-f4596407d8ce.png)
When i change it , Throw an error.

**Comment by ColinWeiss** [comment]
Changing other code doesn't throw an error.

**Comment by ColinWeiss** [comment]
Now,When I Load ExampleMod , it also throw the error.
![image](https://user-images.githubusercontent.com/69356100/95933297-cc946c00-0e00-11eb-91b1-e95adfd45f33.png)

**Comment by ColinWeiss** [comment]
![Uploading Error3.png…]()

**Comment by ColinWeiss** [comment]
Now , when i load ExampleMod , it also throw the error.
![Error3](https://user-images.githubusercontent.com/69356100/95933442-2c8b1280-0e01-11eb-8121-c2e3eaa901ec.png)

**Comment by ColinWeiss** [comment]
And ModName is my ModName...But i is load ExampleMod.

**Comment by ColinWeiss** [comment]
Oh, no, I'm thrown the error when everything I do now！

**Comment by ColinWeiss** [comment]
Well, I found the root of the problem.I can't use the namespace!

**Comment by ColinWeiss** [comment]
![Not Namespace](https://user-images.githubusercontent.com/69356100/95934277-167e5180-0e03-11eb-8b20-9fa572ff1901.png)

**Comment by ColinWeiss** [comment]
![NotNameSpace](https://user-images.githubusercontent.com/69356100/95934459-6fe68080-0e03-11eb-8ba1-690fd77d83cf.png)

**Comment by ColinWeiss** [comment]
![NotNameSpace](https://user-images.githubusercontent.com/69356100/95934476-7bd24280-0e03-11eb-9919-1590a8efec7a.png)

**Comment by ColinWeiss** [comment]
When not have namespace, i can load mod.

**Comment by steviegt6** [comment]
What the? Alright, well I'll have to look into that later. That clears some stuff up, at least.

**Comment by ColinWeiss** [comment]
Thank you very much indeed！！

**Comment by ColinWeiss** [comment]
And , Tea seems that moditem cannot be saved.

**Comment by steviegt6** [comment]
Yeah, I need to figure out how to save modded items as well. Saving is a pain.

**Comment by ColinWeiss** [comment]
I will continue to help you find bugs.  ( ^ v ^ )

**Comment by steviegt6** [comment]
Canceling this project, don't have the incentive to continue.
